multiqc/qualimap/__init__.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from QualiMap """
+
+from __future__ import print_function
+import json
+import logging
+import os
+
+import multiqc
+
+class MultiqcModule(multiqc.BaseMultiqcModule):
+
+    def __init__(self, report):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__()
+
+        # Static variables
+        self.name = "QualiMap"
+        self.anchor = "qualimap"
+        self.intro = '<p><a href="http://qualimap.bioinfo.cipf.es/" target="_blank">QualiMap</a> \
+             is a platform-independent application written in Java and R that provides both \
+             a Graphical User Inteface (GUI) and a command-line interface to facilitate \
+             the quality control of alignment sequencing data and its derivatives like feature counts.</p>'
+        self.analysis_dir = report['analysis_dir']
+        self.output_dir = report['output_dir']
+
+        # Find QualiMap reports
+        qualimap_raw_data = {}
+        for root, dirnames, filenames in os.walk(self.analysis_dir, followlinks=True):
+            if 'genome_results.txt' in filenames and 'raw_data_qualimapReport' in dirnames:
+                with open(os.path.join(root, 'genome_results.txt'), 'r') as gr:
```

Comment:
I've been converting all of the other modules to use the `io` module for better encoding / python 3 support.

_eg:_
```python
with io.open(os.path.join(root, 'genome_results.txt'), 'r', encoding='utf-8') as gr:
```